app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -63,6 +76,10 @@ public void insertMap(MuseumMap map) {
         database.insert(DBConstants.DB_TABLE_MAPS, null, cv);
     }
 
+    /**
+     * Inserts door data to doors table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.